RELEASE.md

```diff
@@ -0,0 +1,71 @@
+# How to make a release
+
+`traefik-proxy` is a package [available on
+PyPI](https://pypi.org/project/jupyterhub-traefik-proxy/).
+The PyPI release is done automatically by TravisCI when a tag
+is pushed.
+
+For you to follow along according to these instructions, you need
+to have push rights to the [traefik-proxy GitHub
+repository](https://github.com/jupyterhub/traefik-proxy).
+
+## Steps to make a release
+
+1. Checkout master and make sure it is up to date.
+
+   ```shell
+   ORIGIN=${ORIGIN:-origin} # set to the canonical remote, e.g. 'upstream' if 'origin' is not the official repo
+   git checkout master
+   git fetch $ORIGIN master
+   git reset --hard $ORIGIN/master
+   # WARNING! This next command deletes any untracked files in the repo
+   git clean -xfd
+   ```
+
+1. Update [changelog.rst](docs/source/changelog.rst) and add it to
+   the working tree.
+   
+   ```shell
+   git add traefik-proxy/docs/source/changelog.rst
+   ```
+
+   Tip: Identifying the changes can be made easier with the help of the
+   [choldgraf/github-activity](https://github.com/choldgraf/github-activity)
+   utility.
+
+1. Set a shell variable to be the new version you want to release.  
+   
+   ```shell
+   VERSION=...  # e.g. 1.2.3
+   git commit -m "release $VERSION"
+   ```
+
+   Tip: You can get the current project version with:
```

Comment:
This is the latest git tag referencing a commit on the branch. I've seen failures where we assume the latest git tag referenced a commit on the branch before, so it is a bit risky to assume this really is the latest released version. Checking the latest tag on github.com is a bit better I think.